src/sortStudents.ts

```diff
@@ -1,16 +1,54 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'desc' | 'asc';
 
+function calcAvGrade(grades: number[]): number {
+  return grades.reduce((x, y) => x + y) / grades.length;
+}
+
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const studentsArr = [...students];
+  const ascOrder = order === 'asc';
+
+  studentsArr.sort((a, b) => {
```

Comment:
useless variable. You can just retiurn result
```suggestion
  const ascOrder = order === 'asc';

  return [...students].sort((a, b) => {
```